subcommands/login/cmd.go

```diff
@@ -28,7 +28,7 @@ func NewCommand() *cobra.Command {
 		Run:   doLogin,
 	}
 	cmd.Flags().BoolVarP(&refreshToken, "refresh-access-token", "", false, "Refresh your current oauth2 access token. This is used when a token's scopes have been updated in app.foundries.io")
-	cmd.Flags().StringVarP(&authURL, "oauth-url", "", client.OauthURL, "OAuth URL to authenticate with")
+	cmd.Flags().StringVarP(&authURL, "oauth-url", "", "", "OAuth URL to authenticate with")
```

Comment:
We need to show some default value to the user.

If the user knows how to override that through the config - that's fine; but I bet that only you or me would ever do that.

There is `Flags.Changed` to verify if some flag was explicitly set by the user.